### PR TITLE
Extract examples

### PR DESCRIPTION
**Pull request by BigBlueHat**
This makes testing/developing/confirming examples (against published contexts) much easier.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,5 +12,16 @@ on the
...
 mailing list as well.
 
+### Testing Examples
+
+The example Verifiable Credentials are available in the [`examples/`](examples/)
+folder and can be tested directly using various tools such as the
+[JSON-LD CLI](https://github.com/digitalbazaar/jsonld-cli).
+
+```sh
+$ npm i -g jsonld-cli
```
Comment (davidlehn) [inline README.md:22]: Could mention the `npx jsonld-cli ...` method too.

File: examples/citizenship.jsonld [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+  },
+  "proof": {
+    "type": "Ed25519Signature2018",
+    "created": "2020-01-30T03:32:15Z",
+    "jws": "eyJhbGciOiJFZERTQSIsI...wRG2fNmAx60Vi4Ag",
+    "proofPurpose": "assertionMethod",
+    "verificationMethod": "did:example:28394728934792387#keys-7f83he7s8"
+  }
+}
+
```
Comment (davidlehn) [inline examples/citizenship.jsonld]:
```suggestion
```

File: examples/ead.jsonld [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+  },
+  "proof": {
+    "type": "Ed25519Signature2018",
+    "created": "2020-01-30T03:32:15Z",
+    "jws": "eyJhbGciOiJFZERTQSIsI...wRG2fNmAx60Vi4Ag",
+    "proofPurpose": "assertionMethod",
+    "verificationMethod": "did:example:28394728934792387#keys-7f83he7s8"
+  }
+}
+
```
Comment (davidlehn) [inline examples/ead.jsonld]:
```suggestion
```

File: .editorconfig [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# EditorConfig is awesome: https://EditorConfig.org
+
+# top-most EditorConfig file
+root = true
+
+# Unix-style newlines with a newline ending every file
+[*]
+end_of_line = lf
+insert_final_newline = true
```
Comment (davidlehn) [inline .editorconfig]: This line about final newlines needs a newline.
```suggestion
insert_final_newline = true
```

**Review by davidlehn — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (BigBlueHat) [reply README.md:22]: Meh. 😉 Primary point is to show it's usable via npm "stuff." Anyone who knows about npx and wants to use that, knows they can.

File: examples/citizenship.jsonld (hunk 2 above)
Comment (BigBlueHat) [reply examples/citizenship.jsonld]: Wait...I thought you _wanted_ trailing line endings... 🤦

File: .editorconfig (hunk 4 above)
Comment (BigBlueHat) [reply .editorconfig]: ...which makes the rest of the "nits" here confusing...just FYI...

File: examples/citizenship.jsonld (hunk 2 above)
Comment (davidlehn) [reply examples/citizenship.jsonld]: Yeah, the final real char and then a newline.  This has a `}`, newline, then a space:
```
$ curl -s https://raw.githubusercontent.com/w3c-ccg/citizenship-vocab/3f3e9a179e4a6889fe331174caf81c67bdade77a/examples/citizenship.jsonld | hd | tail -2
000007c0  73 38 22 0a 20 20 7d 0a  7d 0a 20                 |s8".  }.}. |
000007cb
```

Comment (BigBlueHat) [reply examples/citizenship.jsonld]: This is why not having _any_ whitespace at the end of a file is so nice. 😁 No guesswork. 😉

**Review by msporny — APPROVED** [review]

**Pull request merged**